salt/modules/win_network.py

```diff
@@ -470,3 +516,20 @@ def connect(host, port=None, **kwargs):
     ret['comment'] = 'Successfully connected to {0} ({1}) on {2} port {3}'\
         .format(host, _address[0], proto, port)
     return ret
+
+
+def is_private(ip_addr):
+    '''
+    Check if the given IP address is a private address
+
+    .. versionadded:: 2014.7.0
```

Comment:
Copy/Paste here? This should be `.. versionadded:: Fluorine` instead of these other versionadded and versionchanged tags. :)